printer_mods/jeoje/4.3_Inch_Touchscreen_Mount/README.md

```diff
@@ -0,0 +1,50 @@
+# Voron v2.4 4.3" Touchscreen Mount
+
+Inspired by [Roboticator's excellent 4" touchscreen mount](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/roboticator24/4inch_touchscreen_mount_for_v2.4), I wanted to use a capacitive touchscreen and a DSI connection instead.
```

Comment:
Please change to a relative link:

    [Roboticator's excellent 4" touchscreen mount](../../roboticator24/4inch_touchscreen_mount_for_v2.4)

Reply:
Updated this link and one other in the readme to use relative links